src/Mod/TechDraw/Gui/ViewProviderPage.h

```diff
@@ -149,7 +149,7 @@ class TechDrawGuiExport ViewProviderPage: public Gui::ViewProviderDocumentObject
 private:
     QPointer<MDIViewPage> m_mdiView;
     std::string m_pageName;
-    QGVPage* m_graphicsView;
+    QPointer<QGVPage> m_graphicsView;
     QGSPage* m_graphicsScene;

```

Comment:
as we change the `m_graphicsView` can we also change `m_graphicsScene?` so it is consistent?